releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml

```diff
@@ -0,0 +1,75 @@
+---
+prelude: |
+    The 0.9 release includes new backend options for parallel exeuction
+    of large numbers of circuits on a HPC cluster using a Dask distributed,
+    along with other general performance improvements and bug fixes.
+features:
+  - |
+    Add qiskit library SXdgGate and CUGate to the supported basis gates for
+    the Aer simulator backends. Note that the CUGate gate is only natively
+    supported for the statevector and unitary methods. For other simulation
+    methods it must be transpiled to the supported basis gates for that method.
+  - |
+    Adds support for N-qubit Pauli gate (
+    :class:`qiskit.circuit.library.generalized_gates.PauliGate`) to all
+    simulation methods of the
+    :class:`~qiskit.providers.aer.AerSimulator` and
+    :class:`~qiskit.providers.aer.QasmSimulator`.
+deprecations:
+  - |
+    Passing an assembled qobj directly to the
+    :meth:`~qiskit.providers.aer.AerSimulator.run` method of the Aer simulator
+    backends has been deprecated in favor of passing transpiled circuits
+    directly as `backend.run(circuits, **run_options)`.
+  - |
+    All snapshot instructions in :mod:`qiskit.providers.aer.extensions` have
+    been deprecated. For replacement use the save instructions from the
+    :mod:`qiskit.providers.aer.library` module.
+  - |
+    Adding non-local quantum errors to a
+    :class:`~qiskit.providers.aer.noise.NoiseModel` has been deprecated due to
+    inconsistencies in how this noise is applied to the optimized circuit.
+    Non-local noise should be manually added to a scheduled circuit in Qiskit
+    using a custom transpiler pass before being run on the simulator.
```

Comment:
An example here would be good.

Reply:
I don't have one, and it likely would be too complicated for a release note anyway.